src/templates/person.js

```diff
@@ -1,19 +1,19 @@
 import '@fortawesome/fontawesome-svg-core/styles.css';
 
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 
 import ActionButtons from '../components/action-buttons/action-buttons';
 import Contact from '../components/contact/contact';
 import EventList from '../components/event-list/event-list';
-// import GitHubContributionCalendar from '../components/github-contribution-calendar/github-contribution-calendar';
+import GitHubContributionCalendar from '../components/github-contribution-calendar/github-contribution-calendar';
 import PlayAudio from '../components/play-audio/play-audio';
 import ProfileDescription from '../components/profile-description/profile-description';
 import ProfilePhoto from '../components/profile-photo/profile-photo';
 import PropTypes from 'prop-types';
 import Quote from '../components/quote/quote';
 import SkillsList from '../components/skills-list/skills-list';
 import SocialLinks from '../components/social-links/social-links';
-import { Widget } from 'ssw.rules.widget';
+// import { Widget } from 'ssw.rules.widget';
```

Comment:
Comment the widget back in
```suggestion
import { Widget } from 'ssw.rules.widget';
```